towhee/runtime/pipeline.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+from towhee.runtime.col_storage import ColStorage
+from towhee.runtime.graph import Graph
+
+
+class DataCollection:
+    '''Mock class
+    '''
+    pass
+
+
+class Pipeline:
```

Comment:
can we rename this to something else. The forward facing class is also called pipeline which might get confusing.